SPECS/python-requests-mock/python-requests-mock.spec

```diff
@@ -1,169 +1,94 @@
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
-# Enable python3 build by default
-%bcond_without python3
-# Disable python2 build by default
-%bcond_with python2
-
-%if 0%{?rhel} >= 8
-# Disable tests on epel8 - dependencies dont exist.
-%bcond_with tests
-%else
-%bcond_without tests
-%endif
-
-Name:           python-requests-mock
-Version:        1.7.0
-Release:        3%{?dist}
+%global pypi_name requests-mock
+Name:           python-%{pypi_name}
+Version:        1.8.0
+Release:        6%{?dist}
 Summary:        A requests mocking tool for python
-
 License:        ASL 2.0
+Vendor:         Mariner
+Distribution:   Microsoft Corporation
 URL:            https://requests-mock.readthedocs.io/
-Source0:        https://pypi.io/packages/source/r/requests-mock/requests-mock-%{version}.tar.gz#/python-requests-mock-%{version}.tar.gz
-
+Source0:        https://pypi.io/packages/source/r/requests-mock/requests-mock-%{version}.tar.gz#/%{pypi_name}-%{version}.tar.gz
 Patch0:         0002-Use-system-urllib3-package.patch
 Patch1:         0003-Allow-skipping-purl-tests-if-it-is-not-present.patch
```

Comment:
Please evaluate if these are necessary for Mariner

Reply:
Patch 1 for check section is not needed. Removed.